### PR TITLE
Disable onDragOver activation if effect is not all

### PR DESCRIPTION
**Pull request by voronianski**
When dropping file into browser `effectAllowed` will be `"all"` that's why we can check this in order to skip setting active state on dropzone when it's not necessary, see demo:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,6 +42,14 @@ var Dropzone = React.createClass({
     e.stopPropagation();
     e.dataTransfer.dropEffect = 'copy';
 
+    var effectAllowed = e.dataTransfer.effectAllowed;
+
+    // set active drag state only when file is dragged into
+    // (in mozilla when file is dragged effect is "uninitialized")
+    if (effectAllowed !== 'all' && effectAllowed !== 'uninitialized') {
+      return;
```
Comment (paramaggarwal) [inline index.js]: Please don't `return` here, as we need the `this.props.onDragOver(e)` to still be called always. Instead wrap ``` this.setState({
       isDragActive: true
     });``` inside an `if (effectAllowed === 'all' || effectAllowed === 'uninitialized')`

Comment (voronianski) [inline index.js]: @paramaggarwal updated, thanks!

**Comment by voronianski** [comment]
@paramaggarwal updated

**Comment by paramaggarwal** [comment]
I didn't even know about such an API. Thanks!

**Comment by voronianski** [comment]
@paramaggarwal :wink: thanks!

**Comment by voronianski** [comment]
@paramaggarwal when do you plan to bump npm version?

**Comment by paramaggarwal** [comment]
Done. `v1.2.4`.

**Pull request merged**